charts/tuf/values.yaml

```diff
@@ -15,15 +15,19 @@ deployment:
   port: 8080
 
 secrets:
+  create: true
```

Comment:
Lets default to `false` and then provide testing values so that we can execute the logic that would be applied if necessary

Reply:
Done :)